chrome/content/zotero/xpcom/schema.js

```diff
@@ -477,26 +479,37 @@ Zotero.Schema = new function(){
 			return deferred.promise;
 		})
 		.then(function (updated) {
-			if (runRemoteUpdateWhenComplete) {
-				var deferred = Q.defer();
-				if (updated) {
-					if (Zotero.Prefs.get('automaticScraperUpdates')) {
-						Zotero.proxyAuthComplete
-						.then(function () {
-							Zotero.Schema.updateFromRepository(2, function () deferred.resolve());
-						})
-					}
-				}
-				else {
-					Zotero.proxyAuthComplete
+			if (!runRemoteUpdateWhenComplete) return;
```

Comment:
Ok. Great! We'll wait until then to fix this up. I guess I need to get feed reader merged in before end of the month.